frontend/packages/integration-tests-cypress/tests/crud/image-pull-secret.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { checkErrors, testName } from '../../support';
+import { listPage } from '../../views/list-page';
+import { nav } from '../../views/nav';
+
+const clickCreateImagePullSecretDropdownButton = () => {
+  cy.byTestID('item-create')
+    .click()
+    .get('body')
+    .then(($body) => {
+      if ($body.find(`[data-test-dropdown-menu="image"]`).length) {
+        cy.get(`[data-test-dropdown-menu="image"]`).click();
+      }
+    });
+};
+
+const typeValue = (testId: string, inputValue: string, blur: boolean) => {
+  !blur
```

Comment:
Do we need to call the `blur()` ?
Wont the focus get lost, once next field will get focus to enter the value?